static/collection.js

```diff
@@ -1,47 +1,74 @@
 (async () => {
-    function clear_all_games_table(){
+    function clear_table(){
         const table = document.getElementById('table');
         table.innerText = "";
     }
 
-async function create_all_games_table() {
-    clear_all_games_table();
-    const user = document.getElementById("user").value;
-    fetch('/all_games?user=' + user)
-        .then(async response => {
-            if (response.ok) {
-                const games = await response.json();
-                const table = document.getElementById('table')
-                games.forEach(addGamesToTable)
+    async function create_table(games){
+        const table = document.getElementById('table')
+        const header = table.createTHead();
+        const row = header.insertRow(0);
+        const cell_one = row.insertCell(0);
+        cell_one.innerText = "Game Name (released year)"
+        const cell_two = row.insertCell(1);
+        cell_two.innerText = "Overall Rank"
+        const cell_three = row.insertCell(2);
+        cell_three.innerText = "Average Rating"
+        const cell_four = row.insertCell(3);
+        cell_four.innerText = "Complexity"
+        const cell_five = row.insertCell(4);
+        cell_five.innerText = "Players"
+        games.forEach(addGamesToTable)
 
-                function addGamesToTable(game, index){
-                    const row = table.insertRow(index);
-                    const cell_one = row.insertCell(0);
-                    cell_one.innerText = `${game.name} (${game.yearpublished})`
-                    const cell_two = row.insertCell(1);
-                    cell_two.innerText = `${game.minplayers} - ${game.maxplayers} players`
-                }
+        function addGamesToTable(game, index){
+            const row = table.insertRow(index + 1);
```

Comment:
Must do +1 on index otherwise this will wipe out the header row.  Logically this makes sense but this feels sort of weird to me.  So maybe thats a code sniff there's a better way to add a header to the table.

Reply:
looking at these docs, it looks like you just need to call it without any value. when you add anything in there, it takes it as the index. When you say index 0, its replacing the header at row 0. so maybe try `table.insertRow()` 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableElement/insertRow

Reply:
Ahhhhh yeah makes sense!